prepare_llm_models.sh

```diff
@@ -16,7 +16,7 @@
 #
 
 EMBEDDING_MODEL="thenlper/gte-small"
-if [ -d "$1/facebook/opt-125m" ] && [ -d "$1/$EMBEDDING_MODEL" ]; then
+if [ -d "$1/facebook/opt-125m" ] && [ -d "$1/${EMBEDDING_MODEL}_embeddings" ] && [ -d "$1/${EMBEDDING_MODEL}_tokenizer" ]; then
```

Comment:
Maybe make one directory for all embeddings models to avoid multiple [ ]? Is it planned to include both - 2 output version and 1 output version? @dtrawins @bstrzele